src/services/branch-manager.js

```diff
@@ -1,78 +1,102 @@
-const context = require('@forestadmin/context');
+const Context = require('@forestadmin/context');
 const branchDeserializer = require('../deserializers/branch');
 const EnvironmentSerializer = require('../serializers/environment');
-const { serverHost } = require('../config');
 const { handleError } = require('../utils/error');
 
-const ERROR_MESSAGE_PROJECT_IN_V1 = '⚠️  This project does not support branches yet. Please migrate your environments from your Project settings first.';
-const ERROR_MESSAGE_ENV_SECRET_ISSUE = '⚠️  Your development environment is not properly set up. Please run `forest init` first and retry.';
-const ERROR_MESSAGE_BRANCH_ALREADY_EXISTS = '❌ This branch already exists.';
```

Comment:
Can you explain the reason behind the removal of those error (and success) icons?
I don't get, what is the link with the merge of the Lumber.